unity-renderer/Assets/Scripts/MainScripts/DCL/Components/Textures/Data/TextureModel.cs

```diff
@@ -4,7 +4,7 @@
 using UnityEngine;
 
 [System.Serializable]
-public class TextureModel : BaseModel
+public class TextureModel 
```

Comment:
Now that we are not coupling this models to the components maybe we should relocate them to a hierarchy outside of `DCL/Components`.

Reply:
Yep that is the idea, I was waiting for the Brian PR to move all the new ECS to a new plugin